frontend/src/scenes/onboarding/sdks/sdksLogic.tsx

```diff
@@ -225,8 +257,14 @@ export const sdksLogic = kea<sdksLogicType>([
     afterMount(({ actions }) => {
         actions.loadSnippetEvents()
     }),
-    urlToAction(({ actions }) => ({
-        '/onboarding/:productKey': (_productKey, { sdk }) => {
+    urlToAction(({ actions, values }) => ({
+        '/onboarding/:productKey': (_product_key, { sdk }) => {
+            if (
+                values.productKey === ProductKey.PRODUCT_ANALYTICS &&
+                values.featureFlags[FEATURE_FLAGS.PRODUCT_ANALYTICS_ONBOARDING]
```

Comment:
For an experiment we can do `values.featureFlags[FEATURE_FLAGS.PRODUCT_ANALYTICS_ONBOARDING] === "test"` where "test" is the experiment variant.